src/cluster.h

```diff
@@ -36,5 +32,7 @@ int Cluster_RefreshCluster(RedisModuleCtx *ctx, RedisModuleString **argv, int ar
 int Cluster_ClusterSet(RedisModuleCtx *ctx, RedisModuleString **argv, int argc);
 int Cluster_ClusterSetFromShard(RedisModuleCtx *ctx, RedisModuleString **argv, int argc);
 
-
-#endif /* SRC_CLUSTER_H_ */
+#define VERIFY_CLUSTER_INITIALIZE(c) \
```

Comment:
Why this was moved here?

Reply:
It's cluster-related, seems logical. Do you prefer it like it was?

Reply:
I agree, we can leave it here.